### DISCUSSION TITLE
Cursor glitching

### DISCUSSION THREAD

**Discussion by justmangoou** [ppy/osu#28463] [Q&A] [answered]
### Type

Game behaviour

### Bug description

The cursor randomly stutters while playing on standard (Around 00:02 to 00:03 in the video). 

I tested using OpenTabletDriver outside of osu lazer and didn't see the same problem (I turned off OTD when playing on lazer).
Also, I assume this was caused by the renderer since OpenGL is normal.

### Screenshots or videos

https://github.com/ppy/osu/assets/38831897/a518f9dc-0180-4ddf-92b6-baf5d607e6c6

### Version

2024.521.2-lazer

### Logs

[input.log](https://github.com/user-attachments/files/15812605/1718241644.input.log)
[network.log](https://github.com/user-attachments/files/15812606/1718241644.network.log)
[performance.log](https://github.com/user-attachments/files/15812607/1718241644.performance.log)
[runtime.log](https://github.com/user-attachments/files/15812608/1718241644.runtime.log)
[updater.log](https://github.com/user-attachments/files/15812609/1718241644.updater.log)

**Comment by peppy** [comment]
Maybe probably https://github.com/ppy/osu/issues/19960

**Comment by justmangoou** [comment]
I had looked into this issue before, but disable "High Precision Mouse" didn't fix my problem.  

P/s: I have played using lazer since last year until I noticed this issue and It functioned perfectly fine. There is a possibility it caused by my hardware.

**Comment by justmangoou** [comment, marked as answer]
Update,

It's not because of Osu renderer or OTD, it caused by ducktape.
